radix-engine-common/src/crypto/public_key_ed25519.rs

```diff
@@ -47,7 +47,7 @@ impl TryFrom<&[u8]> for Ed25519PublicKey {
 #[cfg_attr(feature = "radix_engine_fuzzing", derive(Arbitrary))]
 #[derive(Debug, Clone, Copy, PartialEq, Eq, Hash, Sbor)]
 #[sbor(transparent)]
-pub struct Ed25519PublicKeyHash(pub [u8; NodeId::UUID_LENGTH]);
+pub struct Ed25519PublicKeyHash(pub [u8; NodeId::RANDOM_LENGTH]);
```

Comment:
Can we use a different name than `RANDOM_LENGTH`? Sounds like we're choosing a random length.